frontend/src/components/SignUpForm.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { Profile } from './Profile.js';
+
+import { Heading, Text, UserForm, FormLabel, FormInput, Button } from 'styling/styling';
+
+
+const SIGNUP_URL = 'https://agnes-emma-login.herokuapp.com/users';
+
+export const SignUpForm = () => {
+    const accessToken = useSelector((store) => store.user.login.accessToken);
+
+    const [name, setName] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleLoginSuccess = (loginResponse) => {
+    };
+
+    const handleLoginFailed = (loginError) => {
+    };
```

Comment:
This needs to be finished because you leaving your users unlogged, and without any clue, if your app is working or not 😢